apps/src/templates/ChallengeDialog.jsx

```diff
@@ -0,0 +1,139 @@
+import BaseDialog from './BaseDialog';
+import LegacyButton from './LegacyButton';
+import PuzzleRatingButtons from './PuzzleRatingButtons';
+import Radium from 'radium';
+import React from 'react';
+
+const stylesGenerator = assetUrl => ({
+  dialog: {
+    top: '20%',
+    border: '5px solid #75699b',
```

Comment:
Should this be a color in `/shared/css/color.scss`?  Or snap to an existing color?